src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -340,6 +340,11 @@ private void requireDockerApiVersionAtLeast(final String required, final String
     assumeTrue(msg, dockerApiVersionAtLeast(required));
   }
 
+  private boolean storageDriverNot(String driver) throws Exception {
```

Comment:
I would rather see this method changed from `boolean storageDriverNot(String driver)`, which checks the version and returns a `boolean`, to `void requireStorageDriverNotAufs()` which hard-codes a check of the driver against `"aufs"` and passes that to `assumeFalse(...)`. That way in your test, you don't have to wrap everything in `if (storageDriverNot("aufs")) {...}`, your test can just call `requireStorageDriverNotAufs()` and if the test runner sees that the assumption is invalid it will exit the test. See, for example, `requireDockerApiVersionAtLeast(...)` and `requireDockerApiVersionLessThan(...)` which do this same thing.